src/itwinai/torch/distributed.py

```diff
@@ -3,10 +3,14 @@
 import os
 from typing import Any, Callable, Iterable, List, Literal, Optional, Tuple, Union
 
+import deepspeed
```

Comment:
Didn't we talk about putting these imports inside of the place where we use them, i.e. the relevant strategies?

Reply:
True! I moved it down into the `init()` function of the `RayDeepSpeedStrategy`